Tests/StatsdClientTests/Stub|Mock|Fake|Spy/SpyScheme.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  SpyScheme.swift
+//  StatsdClient
+//
+//  Created by Nghia Tran on 10/15/17.
+//  Copyright © 2017 StatsdClient. All rights reserved.
+//
+
+import Foundation
+@testable import StatsdClient
+
+class SpyScheme: FlushScheme {
+
+    var isStartCall = false
```

Comment:
use passive voice for those kind of variables. For instances: `isCallStarted`, `isCallFlushed`